code/game/objects/items/stacks/packagewrap.dm

```diff
@@ -140,9 +140,9 @@
 
 /obj/item/delivery/attack_self(mob/user as mob)
 	user.drop_item(src, user.loc)
-	if(contents.len)
-		if(ishuman(user))
-			user.put_in_hands(contents[1])
+	if(ishuman(user))
```

Comment:
I see that this was already here, but it's probably not a good idea in retrospect
If a monkey or other nonhuman opens the package, nothing will come out
Though I have no idea what `put_in_hands()` does for mobs without hands

Reply:
It *should* just drop the items on the floor for mobs without hands. The crucial point is that it only works on mobs; are there any situations where something that's *not* a mob might try to open a parcel?